frontend/packages/ceph-storage-plugin/src/components/ocs-storage-class-form/ocs-storage-class-form.tsx

```diff
@@ -136,7 +136,7 @@ export const PoolResourceComponent: React.FC<ProvisionerProps> = ({ onParamChang
     return (
       <>
         {!poolDataLoadError && cephClusterObj && (
-          <div className="form-group">
+          <>
```

Comment:
```suggestion
          <div className="form-group"
```

Reply:
oh ok, for now no change required in storage-class-form. got it, i will revert this change